src/azure-cli/azure/cli/command_modules/acr/_constants.py

```diff
@@ -25,6 +25,8 @@
 
 ACR_TASK_QUICKTASK = 'quicktask'
 
+ACR_RUN_DEFAULT_TIMEOUT_IN_SEC = 60 * 60  # 60 minutes
```

Comment:
> DEFAULT_LOG_TIMEOUT_IN_SEC = 60 * 30  # 30 minutes

Why should we increase the timeout?

Reply:
The default server side timeout is 60 minutes and it can be modified. We received a couple of customer issues where the client timed out before the server timeout. So making adjustments to update the default timeout and adjust it based on server side configuration.